### PR TITLE
Update branch for actions

### PR DESCRIPTION
**Pull request by forsyth2**
Update branch for actions. Since `master` was renamed `main`, we need to update the branch used for actions.

### CODE REVIEW

**Comment by forsyth2** [comment]
@tomvothecoder The checks weren't successful and I can't merge this without them being successful. Is there something else that needs to be done? You also mentioned to add the bash commands to handle creating and deleting `gh-pages/index.html`.

File: .github/workflows/build_workflow.yml
Comment (tomvothecoder) [inline .github/workflows/build_workflow.yml:155]:
```suggestion
        uses: ad-m/github-push-action@master
```
The branch reference here shouldn't be updated since it is for another repo.

File: .github/workflows/release_workflow.yml
Comment (tomvothecoder) [inline .github/workflows/release_workflow.yml:64]:
```suggestion
        uses: ad-m/github-push-action@master
```
The branch reference here shouldn't be updated since it is for another repo.

File: .github/workflows/build_workflow.yml
Comment (tomvothecoder) [inline .github/workflows/build_workflow.yml:22]:
```suggestion
        uses: fkirc/skip-duplicate-actions@master
```
The branch reference here shouldn't be updated since it is for another repo.

**Review by tomvothecoder — CHANGES_REQUESTED** [review]
The build should work after my comments are addressed.

After this PR is merged, checkout the `gh-pages` branch and update `index.html` to reference `main`.

**Pull request merged**
